pages/en/anime/watch/[...info].js

```diff
@@ -149,7 +152,10 @@ export default function Watch({
   const [open, setOpen] = useState(false);
   const [isOpen, setIsOpen] = useState(false);
 
-  const { setAutoNext } = useWatchProvider();
+  const [rateModalOpen, setRateModalOpen] = useState(false);
+
+  const { setAutoNext, dataMedia } = useWatchProvider();
+  const { markProgress } = useAniList();
```

Comment:
```suggestion
  const { markProgress } = useAniList(sessions);
```

you need to pass `sessions` to make this hook works